packages/server/lib/helpers/validation.ts

```diff
@@ -4,6 +4,10 @@ export const providerSchema = z
     .string()
     .regex(/^[a-zA-Z0-9_-]+$/)
     .max(255);
+export const providerNameSchema = z
+    .string()
+    .regex(/^[a-zA-Z0-9 _-]+$/)
```

Comment:
I don't think we allow space or underscore in provider name. Only alphanumeric + dash

Reply:
indeed bad c/c